diff --git a/tests/upstream/test/floating-ui-tests/ListboxFocus.tsx b/tests/upstream/test/floating-ui-tests/ListboxFocus.tsx
--- a/tests/upstream/test/floating-ui-tests/ListboxFocus.tsx
+++ b/tests/upstream/test/floating-ui-tests/ListboxFocus.tsx
@@ -16,7 +16,7 @@ interface SelectContextValue {
 const SelectContext = React.createContext<SelectContextValue>({} as SelectContextValue);
 
 /** @internal */
-function Listbox({ children }: { children: React.ReactNode }) {
+function Listbox({ children }: { children: React.OctaneNode }) {
   const [activeIndex, setActiveIndex] = React.useState<number | null>(1);
   const [selectedIndex, setSelectedIndex] = React.useState<number | null>(null);
 
@@ -61,7 +61,7 @@ function Listbox({ children }: { children: React.ReactNode }) {
   );
 
   return (
-    <SelectContext.Provider value={selectContext}>
+    <SelectContext value={selectContext}>
       <button onClick={() => setSelectedIndex(1)} data-testid="reference" type="button">
         Select
       </button>
@@ -70,7 +70,7 @@ function Listbox({ children }: { children: React.ReactNode }) {
           {children}
         </CompositeList>
       </div>
-    </SelectContext.Provider>
+    </SelectContext>
   );
 }
 
