diff --git a/tests/upstream/src/toolbar/group/ToolbarGroup.test.tsx b/tests/upstream/src/toolbar/group/ToolbarGroup.test.tsx
--- a/tests/upstream/src/toolbar/group/ToolbarGroup.test.tsx
+++ b/tests/upstream/src/toolbar/group/ToolbarGroup.test.tsx
@@ -26,11 +26,11 @@ describe('<Toolbar.Group />', () => {
     refInstanceof: window.HTMLDivElement,
     render: (node) => {
       return render(
-        <ToolbarRootContext.Provider value={testToolbarContext}>
-          <CompositeRootContext.Provider value={testCompositeContext}>
+        <ToolbarRootContext value={testToolbarContext}>
+          <CompositeRootContext value={testCompositeContext}>
             {node}
-          </CompositeRootContext.Provider>
-        </ToolbarRootContext.Provider>,
+          </CompositeRootContext>
+        </ToolbarRootContext>,
       );
     },
   }));
