diff --git a/tests/upstream/react/visual/components/Omnibox.tsx b/tests/upstream/react/visual/components/Omnibox.tsx
--- a/tests/upstream/react/visual/components/Omnibox.tsx
+++ b/tests/upstream/react/visual/components/Omnibox.tsx
@@ -204,7 +204,7 @@ export function Main() {
               </div>
               {!hasOptions && <p className="px-4 pb-4">No recent searches.</p>}
               <FloatingList elementsRef={elementsRef}>
-                <SelectContext.Provider value={{activeIndex, getItemProps}}>
+                <SelectContext value={{activeIndex, getItemProps}}>
                   {options.map((option, index) => (
                     <SearchOption
                       key={option}
@@ -229,7 +229,7 @@ export function Main() {
                       }}
                     />
                   ))}
-                </SelectContext.Provider>
+                </SelectContext>
               </FloatingList>
             </div>
           </FloatingFocusManager>
